JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -60,14 +60,16 @@ public async Task DispatchMessage(SQSMessage message, CancellationToken cancella
             }
             catch (MessageFormatNotSupportedException ex)
             {
-                _log.LogTrace($"Didn't handle message [{message.Body ?? string.Empty}]. No serializer setup");
+                _logger.LogTrace("Could not handle message with Id '{MessageId}' because a deserialiser for the content is not configured. Message body: '{MessageBody}'.",
+                    message.MessageId, message.Body);
                 await DeleteMessageFromQueue(message.ReceiptHandle).ConfigureAwait(false);
                 _onError(ex, message);
                 return;
             }
             catch (Exception ex)
             {
-                _log.LogError(0, ex, "Error deserializing message");
+                _logger.LogError(0, ex, "Error deserializing message with Id '{MessageId}' and body '{MessageBody}'.",
```

Comment:
I _think_ there's overloads (unless that's something added in the 2.0.0 version) that allow you to not need to specify the `0` as the first argument.

Reply:
Again consistency, as here it's US English. I think we should use US English consistently.

Reply:
a) I'll check the overloads, but it's almost certainly starting with a `0` to make it compile with version 1.0.

b) Agreed, standardize on US English for consistency :/